Framework/src/TaskRunner.cxx

```diff
@@ -136,10 +137,12 @@ CompletionPolicy::CompletionOp TaskRunner::completionPolicyCallback(gsl::span<Pa
       continue;
     }
 
-    const auto* dataHeader = get<DataHeader*>(input.header.get()->GetData());
+    const auto* dataHeader = CompletionPolicyHelpers::getHeader<DataHeader>(input);
     assert(dataHeader);
 
-    if (!strncmp(dataHeader->dataDescription.str, "TIMER", 5)) {
+    // using matching on DataDescription will avoid string comparison
```

Comment:
Indeed, but this DataDescription is actually "TIMER-" + <task_name>, so I have to compare only 5 first characters.

Reply:
ah ok, than I will remove this commit, thought it was a full match